hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenCreateHandler.java

```diff
@@ -211,7 +211,9 @@ private Token buildToken(
                 false,
                 op.tokenType(),
                 op.supplyType(),
-                resolvedExpiryMeta.autoRenewNum(),
+                AccountID.newBuilder()
+                        .accountNum(resolvedExpiryMeta.autoRenewNum())
+                        .build(),
```

Comment:
ExpiryMeta should have `AccountID` instead.

Reply:
I started to change this and realized there are lots of code related to this change, and some mono tests as well. Rolled back the changes and created an issue #7350 and assigned it to me after this pr merged.

Reply:
Thanks Iris